### DISCUSSION TITLE
关于二维码解码

### DISCUSSION THREAD

**Discussion by XinSSS** [he3-app/he3-feedback#50] [Ideas]
目前只有文件上传解码， 希望能够有获取剪贴板的方法
 比如我已经复制了一张二维码图片， 在程序中ctrl+v就可以直接识别

**Comment by hongchaodeng** [comment]
你好，你说的二维码解码是指哪个工具？我不太理解，能截个图吗？

**Reply by XinSSS** [reply]
![Snipaste_2023-03-13_16-06-16](https://user-images.githubusercontent.com/30732527/224642218-6cd17dff-bf5a-4e5f-a3db-6ad537d1ca71.png)

**Comment by zhenweiwang1990** [comment]
这里应该增加一个能力就是可以直接用 ctrl+v 来粘贴图片。

**Comment by zhoujunxuan** [comment]
或者类似shareX的扫描屏幕二维码的功能
![image](https://user-images.githubusercontent.com/37800586/229536688-f152311f-f001-447f-9956-7d6b68e180c1.png)

**Reply by Margular** [reply]
顶，同求

**Reply by lyzhang1999** [reply]
https://github.com/he3-app/he3-feedback/issues/255
